rudder-lang/src/parser.rs

```diff
@@ -600,9 +600,24 @@ fn pstruct(i: PInput) -> PResult<HashMap<String, PValue>> {
     )(i)
 }
 
+/// Alternative version of pstruct based on "." rather than braces.
+/// Used for the agent
+/// A struct is stored in a HashMap
+// fn pstruct_agent(i: PInput) -> PResult<HashMap<String, PValue>> {
```

Comment:
Why is this here ?

Reply:
I have to manually remove it from every curent PR since it is the same code i rebased on last week